### DISCUSSION TITLE
LFX Mentorship Spring 2022 Mentee Applications

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#478] [Announcements]
Hello everyone,

The LFX mentorship projects are now listed publicly, and can be found here:
https://github.com/cncf/mentoring/blob/main/lfx-mentorship/2022/01-Spring/README.md

~Unfortunately, we are having an issue with our system, and it may take some time before the projects are open for applications.~

We will update this thread with a timeline as we get more information.

Applications are now open! Please let us know if you run into any issues applying.

# Timeline

## Spring Term: March 1st - May 31st

* Jan 26th - Feb 1st, 2022 (1 week): Project applications open
* Feb 2nd - Feb 13th, 2022 (2 weeks): Mentee applications open
* Feb 15th - Feb 25th, 2022: Application review by the mentors/admission decisions/HR paperwork

## Mentorship duration - three months (12 weeks - full-time schedule)

* March 1, 2022 (Week 1): Mentorship program begins with the initial work assignments
* April 12, 2022 (End of Week 6): Midterm mentee evaluations and first stipend payments
* May 31, 2022 (End of Week 12): Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals

**Comment by nate-double-u** [comment]
Application are now open!

**Comment by nate-double-u** [comment]
A note: currently, some listings don’t appear to have mentors attached (or visible). We’re working on this, and it shouldn’t affect the application process. Mentors will see all applicants when the project administrators add them.

**Comment by nate-double-u** [comment]
A friendly reminder that the mentee applications window is closing on Feb 13th, 2022. If you're interested in being a mentee, get your applications in!

**Reply by nvarun** [reply]
Are working professionals allowed to participate as mentees? How much time must be spent on the project?

**Reply by nate-double-u** [reply]
Hi @nvarun, here are the eligibility and time requirements for the program:
* https://docs.linuxfoundation.org/lfx/mentorship/mentee-guide/am-i-eligible
* https://docs.linuxfoundation.org/lfx/mentorship/mentorship-program-timelines

**Reply by nate-double-u** [reply]
Regarding time spent on the projects, it's expected that a mentee would be working full time for the project.

If a professional had time to participate, I suppose they could apply, provided they don't run afoul of the clause that they
> Not be subject to any existing obligations to third parties (such as contractual obligations to an employer) that would restrict or prohibit your participation in a mentorship program.

**Reply by nvarun** [reply]
Thanks @nate-double-u

**Comment by nate-double-u** [comment]
Applications are now closed for the Spring 2022 LFX Mentorship cycle. Thanks for applying, everyone!

**Reply by RainbowMango** [reply]
As a mentor, I have a question about how to submit the final decision on the system?

**Reply by RainbowMango** [reply]
I can see the mentee application status according to [ManageMenteesApplications](https://docs.linuxfoundation.org/lfx/mentorship/administrators/manage-mentees-applications) but I can't change the application status.

**Reply by nate-double-u** [reply]
Hi @RainbowMango, on Feb 15, 2022 @idvoretskyi sent out an email with the subject line: **Form shared with you: "LFX Mentorship - Spring'2022 mentees acceptance form"** -- it's a link to a form to use for selection submissions. Please let us know if you haven't received it.

**Reply by RainbowMango** [reply]
Thanks @nate-double-u, I got the form now.

**Comment by nate-double-u** [comment]
We have entered about 2/3rds of the selected applicants into the system. We’re still working on getting the remaining candidates in, please be patient as we go through this process. Sorry about the delay.
